proto/ibc/lightclients/tendermint/v1/tendermint.proto

```
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package ibc.lightclients.tendermint.v1;
+
+option go_package = "github.com/cometbft/cometbft/proto/ibc/lightclients/tendermint/v1";
+
+import "tendermint/types/validator.proto";
```

buf (1.47.2)

import "tendermint/types/validator.proto": file does not exist

(COMPILE)